### ISSUE TITLE
can't package the plugin into the project with unreal 4.19.2

### ISSUE THREAD

**Issue by gouzibaobao** [getnamo/leap-ue4#27]
i use the unreal 4.19.2 to make a project with default leap motion plugin in engine.
everything works well  in editor
but ,i can not package the plugin into my project,event there is no error when package the project.
i can not find the hands in packaged project 
C++class was tried ,several ways i find also tried ,but no result.
can there be any ways to solve this problem?
@getnamo  you mentioned that the engine's default plugin is already abandoned, there is any ways to solve my problem?except to use the lastest released project~
wish your answer,
since i am  a  new developer ,a clear and easy ways is very needed.
thanks very much

**Comment by getnamo** [comment]
I doubt patching the plugin for 4.19 will be easy/trivial. 4.19 uses [IWYU](https://docs.unrealengine.com/en-us/Programming/UnrealBuildSystem/IWYUReferenceGuide) exclusively, which this plugin doesn't use (didn't have IWYU back when the plugin was merged). This means many many headers need to be fixed for a successful package. Unless you have strong C++ skills and patience, your best bet is to use the new plugin and port your changes to it. It will also make your project future proof.

**Comment by gouzibaobao** [comment]
yes ,you are right,the project really notice me many cpps need to include headers ,it makes me puzzle~
now ,i have to use the new plugin to do all the functions.
maybe ,we should tell the unreal engine to change to the lastest realesed plugins.
oh,no,it is a whole day to fight !

**Comment by gouzibaobao** [comment]
@getnamo  is it have another solve way for v2 plugin ? 
my project have so manythings to change ,and the new plugins event and interface are too different for me to change .
i just want to package successful 
if i have time , i can change to new plugin, but now ,it is no time to do so~
god~

**Comment by gouzibaobao** [comment]
@getnamo  i have already to change to use the lastest plugin. Thank you ,hope the epic refresh this Default Plugin to New~

**Comment by gouzibaobao** [comment]
I think it is the IWYU made the Plugin can not package successful. after change to new Plugin， I will check whether it works  with old engine default LeapMotion Plugin

**Comment by gouzibaobao** [comment]
@getnamo  the latest new plugin is working ,i have already packaged successful,~now i will try to package IWYU ~hope that will done~

**Comment by z3099644** [comment]
@getnamo, in another post you said that there might be an increased interest in back porting the latest V3 plugin for use on ue version 4.19.2. How are things looking on that front?

We are at a stage where we are ready to ship but it's failing the packaging process. I first tried using a modified 3.2.0 plugin, then reverted back to a 2.17 plugin but neither worked.

If there isn't much hope for a back port we will probably have to migrate our project to use 4.22.

**Comment by getnamo** [comment]
It is highly recommended to migrate the project to use 4.21 or newer (4.22 being ideal at this time)
